flags.go

```diff
@@ -84,6 +121,17 @@ func Enabled(ctx context.Context, name string) (enabled bool) {
 		stats.Gauge("goforit.flags.enabled", gauge, []string{fmt.Sprintf("flag:%s", name)}, .1)
 	}()
 
+	defer func() {
+		updateMtx.RLock()
+		staleness := time.Since(lastUpdate)
```

Comment:
What happens if we're not using an UpdatableBackend?

Reply:
This is separate from the backend being Updatable. lastUpdate tracks the last time the goforit client ran the refreshflags cycle. I can see how the naming is confusing here, I will rename this var.